### DISCUSSION TITLE
WiX Burn Bundle V4: Start Browser with same rights as Installer

### DISCUSSION THREAD

**Discussion by sude22** [wixtoolset/issues#7761] [Questions]
Hi,
we use a WiX Burn Bundle V4 to deploy our app. The license aggrement is a local html file set with
`<bal:WixStandardBootstrapperApplication
	LicenseUrl="LicenseAgreements_en.html"`

When the installer is started with administrator rights, the files are placed in "C:\Windows\Temp\\{UUID}" by default. When clicking on the license URL, the default browser is started without admin rights and has no access to the license file.
If the browser would be started with the same rights as the installer the problem would be solved.

Changing the temporary directory with the commandline parameter "-burn.engine.working.directory" or the registry key "HKLM\Software\Policies\Wix\Burn\EngineWorkingDirectory" would mean to deliver an additional application or batch file, which we want to avoid.

**Comment by grafanauser** [comment]
Same issue here.

**Reply by sude22** [reply]
I hope that this question has not been forgotten. An answer from the developers would be very welcome.

**Comment by chrpai** [comment]
Why does your EULA need to be elevated?  Is it an app or something?  EULAs are typically read only documents in that scenario the LicenseFile (RTF not HTML) would be the simpler way to display a EULA.

**Reply by sude22** [reply]
The EULA file must be available as a local HTML file, as it is too complex to be stored in just one RTF file.
The problem is the display of the EULA HTML file with the default browser. If the installer is started with administrator rights, the HTML file is copied to C:\Windows\Temp\{UUID} by the installer. The files there can only be opened with administrator rights. If the user then clicks on the EULA link in the installer, the standard browser is started without admin rights and cannot access the HTML file under C:\Windows\Temp\{UUID} and an error message appears.

**Reply by chrpai** [reply]
Why not just host the HTML on a website?

**Reply by sude22** [reply]
Unfortunately, this is not an option.

Another solution would be to copy the HTML file as default to a temporary location that does not require administrator rights to read.

**Reply by sude22** [reply]
The problem has still not been resolved. I am now trying one last time to bring it into focus here. It is a great pity that such a trivial problem cannot be answered by one of the experts. If such a service is only available to FireGiant customers, please let us know.

**Reply by robmen** [reply]
You are free to ask questions here, but that doesn't mean you are entitled to answers. If you want guaranteed answers to your questions, then you want a [support contract](https://www.firegiant.com/services/support-programs/).

Speaking personally, this topic wasn't interesting enough for me to spend _my free time_ on it. I chose to go kick the soccer ball around with my kids instead.

**Comment by chrpai** [comment]
I'm kind of in the same camp. I volunteer in the FOSS world by contributing, maintaining a few projects and answer questions when I can.  I'm somewhat hampered by the fact that my employer has an org in GHEC so context switching is really a pain.  Normally when I pop in here it just says I can't contribute because it's outside of my enterprise and that friction alone is enough to move on most of the time.

If you could create a minimum reproducible example and share it via a github repos I'd me more inclined to look at it.  Otherwise the whole scenario isn't clear to me and sounds suspect since generally running a browser elevated is a bad thing.

I also provide professional consulting services if you really need an answer.
